index.bs

```diff
@@ -788,6 +789,32 @@ the {{makeCredential()}} or {{getAssertion()}} method. It mirrors the fields of
 </div>
 
 
+### Credential Transport enumeration (enum <dfn enum>ExternalTransport</dfn>) ### {#transport}
+
+<pre class="idl">
+    enum ExternalTransport {
+        "USB",
+        "NFC",
+        "BLE"
+    };
```

Comment:
in https://fidoalliance.org/specs/fido-u2f-v1.0-nfc-bt-amendment-20150514/fido-u2f-javascript-api.html the Transport enum is..
```
enum Transport {
    "bt",
    "ble",
    "nfc",
    "usb"
};
```
..any reason for webauthn's ExternalTransport to differ? ie, why not also have "bt", and perhaps it'll be better for deployers to also use lowercase for enum members since that's what U2F is using?